src/components/state/Process/process.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import './style.css'
+import productivity2 from '../../../resources/img/work-imgs/productivity-1.svg'
+import arrowRight from '../../../resources/img/work-imgs/arrowRight.svg'
+import arrowLeft from '../../../resources/img/work-imgs/arrowLeft.svg'
+class Process extends Component {
+    state = {  }
+    render() { 
+        return ( 
+        <div className = "process">
+            <div className="headers">
+                <h4 className="processMiniHeader">PROCESS</h4>
+                <h1 className = "methodMadnessHeader">The method to our madness</h1>
```

Comment:
This isn't centered: 
![image](https://user-images.githubusercontent.com/25378966/46922900-954a1500-cfc4-11e8-872b-b737a360d54d.png)